frontend/ui/widget/sortwidget.lua

```diff
@@ -408,7 +408,8 @@ function SortWidget:_populateItems()
             item
         )
     end
-    self.footer_page:setText(T(_("Page %1 of %2"), self.show_page, self.pages), self.footer_center_width)
+    -- NOTE: We forgo our usual "Page x of y" wording because of space constraints given the way the widget is currently built
+    self.footer_page:setText(T(_("%1 / %2"), self.show_page, self.pages), self.footer_center_width)
```

Comment:
That probably requires a context since it could also mean completely different things.

Reply:
Done, under `Pagination`, as I didn't find anything obviously relevant, and it could come in handy down the line, so I went fairly generic.